tests/utils.py

```diff
@@ -290,30 +288,58 @@ def _test_script(
         )
 
 
-def _test_save(script_dir):
-    """Test the save script
-
-    Args
-    ----
-        script_dir: The directory containing the script to test
-
-    """
-    # Running multiple tests at the same time.
-    custom_image_tag = f"test-{uuid.uuid4()}"
-
-    pattern = str(script_dir / f"{custom_image_tag}_*.tar.gz")
-    matching_files = glob.glob(pattern)
-
-    assert len(matching_files) == 0
-
+@contextmanager
+def mocked_binaries():
+    """Mock the binaries in the PATH to avoid computationally intensive operations during testing."""
     mocks_bin = RESOURCES_PATH / "mocks" / "bin"
     current_path = os.environ.get("PATH", "")
     extended_path = f"{mocks_bin}:{current_path}"
 
     with patch.dict("os.environ", PATH=extended_path):
-        _test_script(
-            script_path=script_dir / "do_save.sh",
-            extra_arg=custom_image_tag,
-        )
+        yield
+
+
+@contextmanager
+def zipfile_to_filesystem(output_path=None):
```

Comment:
And also just for my understanding: the option to leave `output_path` empty is only there for some of the tests, right?

Reply:
Indeed. Actually think it might be cleaner to have those tests just use a custom zipfile handle.